mm2src/coins/eth/web3_transport.rs

```diff
@@ -25,13 +27,25 @@ fn single_response<T: Deref<Target = [u8]>>(response: T) -> Result<Json, Error>
     }
 }
 
-#[derive(Debug, Clone)]
+#[derive(Clone)]
 pub struct Web3Transport {
     id: Arc<AtomicUsize>,
     uris: Vec<http::Uri>,
+    event_handlers: Vec<RpcTransportEventHandlerShared>,
+}
+
+impl fmt::Debug for Web3Transport {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("Web3Transport")
+            .field("id", &self.id)
+            .field("uris", &self.uris)
+            // skip the self.event_handlers
```

Comment:
I think we might be interested to see what event handlers are connected to the Web3Transport and other RPC clients. `+ Debug` for RpcTransportEventHandlerShared?